frontend/package.json

```diff
@@ -43,6 +43,7 @@
   },
   "dependencies": {
     "URIjs": "^1.14.1",
+    "angular-elastic": "^2.5.0",
```

Comment:
@oliverguenther this is not installed upon running ```npm install``` and I guess it has to do with shrinkwrap.